src/stylesheets/components/_concept.scss

```diff
@@ -281,3 +288,104 @@
     width: 0;
   }
 }
+
+.demo {
+
+  #msg-one {
+    opacity: 0;
+  }
+
+  #msg-two {
+    opacity: 0;
+  }
+
+  #msg-three {
+    opacity: 0;
+  }
+
+  #msg-four {
+    opacity: 0;
+  }
+
+  #msg-five {
+    opacity: 0;
+  }
+
+  #msg-six {
+    opacity: 0;
+  }
+
+  #msg-seven {
+    opacity: 0;
+  }
+
+  #msg-eight {
+    opacity: 0;
+  }
+}
+
+.messages-appearing.messages {
+
+  overflow: visible;
+  z-index: 2;
+
+  #msg-one {
+    opacity: 1;
+    transition: opacity 1s;
+
+    span::before {
+      opacity: 0;
+      transition: opacity 0.5s 5s;
+    }
+  }
+
+  #msg-two {
+    opacity: 1;
+    transition: opacity 1s 3s;
+  }
+
+  #msg-three {
+    opacity: 1;
+    transition: opacity 1s 5s;
+
+    span::before {
+      opacity: 0;
+      transition: opacity 0.5s 9s;
+    }
+  }
+
+  #msg-four {
+    opacity: 1;
+    transition: opacity 1s 9s;
+
+    span::before {
+      opacity: 0;
+      transition: opacity 0.5s 13s;
+    }
+  }
+
+  #msg-five {
+    opacity: 1;
+    transition: opacity 1s 13s;
+  }
+
+  #msg-six {
+    opacity: 1;
+    transition: opacity 1s 15s;
+  }
+
+  #msg-seven {
+    opacity: 1;
+    transition: opacity 1s;
+
+    span::before {
+      opacity: 0;
+      transition: opacity 0.5s 3s;
+    }
+  }
```

Comment:
if you wanted to be fancy you could come up with some mixin or function to output all of this. You could make it shorter and easier to update if you needed to change the timings, but def not necessary